gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/weapon/identification/epic/wither.json

```diff
@@ -171,34 +173,32 @@
               {
                 "text": " "
               },
-              [
-                {
-                  "score": {
-                    "name": "$atkspeed_stored_10",
-                    "objective": "gm4_ce_data"
-                  },
-                  "color": "dark_green",
-                  "italic": false
-                },
-                {
-                  "text": ".",
-                  "color": "dark_green",
-                  "italic": false
-                },
-                {
-                  "score": {
-                    "name": "$atkspeed_stored_01",
-                    "objective": "gm4_ce_data"
-                  },
-                  "color": "dark_green",
-                  "italic": false
-                }
-              ],
               {
-                "text": " "
-              },
-              {
-                "translate": "attribute.name.generic.attack_speed",
+                "translate": "attribute.modifier.equals.0",
+                "with": [
+                  [
+                    {
+                      "score": {
+                        "name": "$atkspeed_stored_10",
+                        "objective": "gm4_ce_data"
+                      },
+                      "color": "dark_green",
+                      "italic": false
```

Comment:
This color and italic is only set for the first value before the decimal point. I guess in this case it doesn't actually matter since it's wrapped around the same color and italics, but to make it work for other instances (i.e. when the color of the value is not the same as the modifier text), the color and italic should be set after line 196

Reply:
trying to put the formatting after line 196 doesn't work. I think you can only set it per object or at the very end. In any case (also for the next comment) it seems to work as it is

Reply:
Alright, if it works, it works. Hopefully minecraft doesn't change the way they handle the json.